internal/xds/bootstrap/bootstrap.yaml.tpl

```diff
@@ -23,6 +23,33 @@ dynamic_resources:
     ads: {}
     resource_api_version: V3
 static_resources:
+  listeners:
+  - name: envoy-gateway-proxy-ready-{{ .ReadyServer.Address }}-{{ .ReadyServer.Port }}
+    address:
+      socket_address:
+        address: {{ .ReadyServer.Address }}
+        port_value: {{ .ReadyServer.Port }}
+        protocol: TCP
+    filter_chains:
```

Comment:
lets also add a text based access log that outputs to stdout https://github.com/envoyproxy/gateway/blob/75e7ada61fe1db9455481f2321309e099f3d8977/internal/xds/translator/listener.go#LL57C19-L57C19

Reply:
as well as the per connection setting https://github.com/envoyproxy/gateway/blob/75e7ada61fe1db9455481f2321309e099f3d8977/internal/xds/translator/listener.go#L61

Reply:
that will be too noise

Reply:
@psinghal20 can you incorporate the `PerConnectionBufferLimitBytes` setting into the config, to reduce buffer space, tia !